CONTRIBUTING.md

```diff
@@ -93,6 +93,23 @@ for more information.
   commit message title. To a reasonable extent, additional tags can be used
   to signify the component changed, e.g.: `[PI]`, `[CUDA]`, `[Doc]`.
 
+### Pre-commit checks
+
+- Before submitting your patch you can verify coding-style conformance with the clang-format tool:
+  ```
+  git diff -U0 sycl~0 --no-color -- | ./clang/tools/clang-format/clang-format-diff.py -p1
+  ```
+  make sure clang-format is installed on your system
```

Comment:
Could this be merged with lines 81-84?

Reply:
you are right! sorry.
i will integrate it into what's there already. just thought we could provide a lookup for pre-commit commands.

Reply:
Sure, the commands are really useful.